api/src/main/resources/liquibase.xml

```diff
@@ -1,9 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@teggsdgreat please fix this, you shouldn't delete everything in this file, only add your new change